tests/modules/images/webp-uploads/rest-api-tests.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Tests for webp-uploads module rest-api.php.
+ *
+ * @package performance-lab
+ * @group   webp-uploads
+ */
+
+use PerformanceLab\Tests\TestCase\ImagesTestCase;
+
+class WebP_Uploads_REST_API_Tests extends ImagesTestCase {
```

Comment:
I don't think extending for `ImagesTestCase` is either required or necessary here, since those assertions are not used in this class.


```suggestion
class WebP_Uploads_REST_API_Tests extends WP_UnitTestCase {
```